index.html

```diff
@@ -13,7 +13,20 @@
 
 <body>
     <div class="container">
-   
+    <header class="top-nav-bar">
+        <div class="top-flex-item"><img src=".\img\lambda-black.png"></div>
```

Comment:
when using the <img /> tag it is always best practice to use the "alt" attribute to help with accessibility. "alt" is just a description of the expected image to be displayed it is great with accessibility and SEO